### ISSUE TITLE
Adjust position of plots for Attack seq

### ISSUE THREAD

**Issue by alosdiallo** [alosdiallo/HiC_Network_Viz_tool#4]
Adjust the positions of the plots so that you don't have to scroll to view them.

**Comment by KHBHH** [comment]
I am working on this, using the CSS trick.

**Comment by alosdiallo** [comment]
No problem

**Comment by KHBHH** [comment]
I tried to adjust the size of plots using the CSS trick, so that they will fit in one panel (I have inlcuded the UI in html file), you can test it here:
https://hmsimmunodept.shinyapps.io/DNARchitect_qa/

Now, The buttons show beside each other above the plots whenever I select more than a plot, for instance(HiC-ATAC).
Before, I had the download button under each plot. What do you think?

**Comment by KHBHH** [comment]
I will make the fontsize of texts on the plot bigger (I will inlcude it later)

**Comment by KHBHH** [comment]
This is done, but still need to deploy it to shinyapps.io to make sure it works there, will let you know till tomorrow!

**Comment by KHBHH** [comment]
I deployed it to shinyapps.io, you can check it here. https://hmsimmunodept.shinyapps.io/DNARchitect_qa/

I changed the size of the plots (they look a little bit smaller now) and if more than 1 data is chosen, they should show beneath each other without so much space between them. 

Right now I just need to check if this looks fine in smaller screens, as my PC has a bigger one.  Please let me know what you think.

**Comment by KHBHH** [comment]
I can change the sizes of the plots if they look small.

**Comment by KHBHH** [comment]
I fixed the 'croping genome names' problem, but I need to test it on smaller screen to make sure it is working. Mine is pretty large.

**Comment by KHBHH** [comment]
Regarding this, I tried to play around with the css trick, but it seems that the extra space in the plot is from the plot itself, and it makes sense. With hic_bedpe file the genes underneath the plot are few, that's why we notice the white space. But when I did plot K562_NHEK_GM12878-loops.txt the genes filled the space. Maybe it is better to discuss this live.

![hic_bedpe](https://user-images.githubusercontent.com/39968446/55461696-5a30b200-55f5-11e9-94b9-ebb9262f0a2c.png)

(the file attached below does not show bezier curves, that is another issue that I will check with Ricardo, I just put it so that the genes are shown)

![hg19-pax5](https://user-images.githubusercontent.com/39968446/55461396-9fa0af80-55f4-11e9-8d0c-809dfdc02bbe.png)
